gui/src/app/view/psbt.rs

```diff
@@ -151,15 +159,50 @@ pub fn broadcast_action<'a>(warning: Option<&Error>, saved: bool) -> Element<'a,
             Column::new()
                 .spacing(10)
                 .push_maybe(warning.map(|w| warn(Some(w))))
-                .push(text("Broadcast the transaction"))
+                .push(Container::new(h4_bold("Broadcast the transaction")).width(Length::Fill))
+                .push_maybe(if conflicting_txids.is_empty() {
+                    None
+                } else {
+                    Some(
+                        Row::new()
+                            .push(icon::warning_icon())
+                            .push(text(format!(
+                                "WARNING: The following transaction{} spending one or more \
+                                inputs from the transaction to be broadcast and will be \
+                                dropped, along with any other transactions that depend on {}:",
```

Comment:
Let's have a higher level first sentence before the technical transaction dependency description? I'm thinking something along the lines of "WARNING: replacing this transaction would invalidate some later payments. [...]".